### PR TITLE
Ensure that send_key doesn't run on virtio console

### PR DESCRIPTION
**Pull request by asdil12**
Call show_tasks_in_blocked_state after selecting
non-virtio console.

https://progress.opensuse.org/issues/39497

### CODE REVIEW

File: lib/console_yasttest.pm [hunk 1]
```diff
@@ -9,11 +9,11 @@ use utils 'show_tasks_in_blocked_state';
 sub post_fail_hook {
     my $self = shift;
 
-    show_tasks_in_blocked_state;
-
     select_console 'log-console';
     save_screenshot;
 
+    show_tasks_in_blocked_state;
```
[the remark this answers is not in the document]
Comment (asdil12) [reply lib/console_yasttest.pm]: I could alternatively only run `show_tasks_in_blocked_state` unless `testapi::is_serial_terminal()`.

[the remark this answers is not in the document]
Comment (asdil12) [reply lib/console_yasttest.pm]: Test for old patch (to be replaced in a minute): http://artemis.suse.de/tests/480

**Comment by asdil12** [comment]
Test for new patch: http://artemis.suse.de/tests/481

File: lib/console_yasttest.pm [hunk 2]
```diff
@@ -9,11 +9,21 @@ use utils 'show_tasks_in_blocked_state';
...
+        show_tasks_in_blocked_state;
+    }
 
     select_console 'log-console';
     save_screenshot;
 
+    if ($defer_blocked_task_info) {
+        show_tasks_in_blocked_state;
+    }
+
```
Comment (okurz) [inline lib/console_yasttest.pm]: How about – personal taste:

```perl
my $defer_blocked_task_info = testapi::is_serial_terminal() ? 1 : 0;
show_tasks_in_blocked_state unless $defer_blocked_task_info;
…
show_tasks_in_blocked_state if $defer_blocked_task_info;
```

**Comment by asdil12** [comment]
http://artemis.suse.de/tests/489

**Pull request merged**